test/functional/feature_on_chain_government_voting_scenarios.py

```diff
@@ -13,13 +13,14 @@
 APPROVAL_THRESHOLD=50
 QUORUM=50
 VOTING_PERIOD=10
+NEXT_NETWORK_UPGRADE_HEIGHT=200
 
 class OCGVotingScenarionTest(DefiTestFramework):
     def set_test_params(self):
         self.num_nodes = 1
         self.setup_clean_chain = True
         self.extra_args = [
-            ['-jellyfish_regtest=1', '-dummypos=0', '-txnotokens=0', '-amkheight=50', '-bayfrontheight=51', '-eunosheight=80', '-fortcanningheight=82', '-fortcanninghillheight=84', '-fortcanningroadheight=86', '-fortcanningcrunchheight=88', '-fortcanningspringheight=90', '-fortcanninggreatworldheight=94', '-grandcentralheight=101', '-simulatemainnet=1'],
+            ['-jellyfish_regtest=1', '-dummypos=0', '-txnotokens=0', '-amkheight=50', '-bayfrontheight=51', '-eunosheight=80', '-fortcanningheight=82', '-fortcanninghillheight=84', '-fortcanningroadheight=86', '-fortcanningcrunchheight=88', '-fortcanningspringheight=90', '-fortcanninggreatworldheight=94', '-grandcentralheight=101', '-nextnetworkupgradeheight={0}'.format(NEXT_NETWORK_UPGRADE_HEIGHT), '-simulatemainnet=1'],
```

Comment:
A bit cleaner this way IMO. 

```suggestion
            ['-jellyfish_regtest=1', '-dummypos=0', '-txnotokens=0', '-amkheight=50', '-bayfrontheight=51', '-eunosheight=80', '-fortcanningheight=82', '-fortcanninghillheight=84', '-fortcanningroadheight=86', '-fortcanningcrunchheight=88', '-fortcanningspringheight=90', '-fortcanninggreatworldheight=94', '-grandcentralheight=101', f'-nextnetworkupgradeheight={NEXT_NETWORK_UPGRADE_HEIGHT}', '-simulatemainnet=1'],
```

Reply:
Yeah, I agree. I saw it could be done like that too (I'm not so up-to-date with Python syntax).